lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -438,6 +483,20 @@ bool SessionState::apply_update_criteria(
     gy_dynamic_rules_.remove_rule(rule_id, nullptr);
   }
 
+  // Converged 5G rules to install
+  for (const auto& rule : session_uc.pdrs_to_install) {
+    int32_t Pdr_index;
+    Pdr_index = pdr_index(rule.pdr_id());
+    if (Pdr_index != -1) {
+      // Update the existing value
+      PdrList_.at(Pdr_index) = rule;
```

Comment:
this seems like a map interface? Why don't we change the type of `PdrList_` to a map?

Reply:
PdrList_ as a vector of SetGroupPDR